### PR TITLE
Include the custom domain URL in the scripts

### PR DESCRIPTION
**Pull request by urvishp80**
Added `https://mailing-list.bitcoindevs.xyz/bitcoindev/` in the list for all the scripts

### CODE REVIEW

File: generate_weekly_newsletter_json.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
     elastic_search = ElasticSearchClient()
     dev_urls = [
         ["https://lists.linuxfoundation.org/pipermail/bitcoin-dev/",
-         "https://gnusha.org/pi/bitcoindev/"],
+         "https://gnusha.org/pi/bitcoindev/",
```
Comment (kouloumos) [inline generate_weekly_newsletter_json.py:23]: What is the reason that we are keeping `https://gnusha.org/pi/bitcoindev` here? 
This also extends to `https://lists.linuxfoundation.org/pipermail/bitcoin-dev/` which is in the above line.

Comment (urvishp80) [reply generate_weekly_newsletter_json.py:23]: Keeping `https://gnusha.org/pi/bitcoindev` only as a fallback approach. 

And for `https://lists.linuxfoundation.org/pipermail/bitcoin-dev/`, if we are sure that there won't be any new thread there, we can remove it.

Should I remove them both?

Comment (kouloumos) [reply generate_weekly_newsletter_json.py:23]: > Keeping https://gnusha.org/pi/bitcoindev only as a fallback approach.

what fallback means in this case?

When we scrape the mailing list we scrape just `https://mailing-list.bitcoindevs.xyz/bitcoindev/`, right?

Comment (urvishp80) [reply generate_weekly_newsletter_json.py:23]: The workflow is like this - 
In Scraper Repo:
1. Scrape the docs from `https://gnusha.org/pi/bitcoindev` and insert them into ES
2. While inserting these docs the change the `domain` URL from `https://gnusha.org/pi/bitcoindev` to `https://mailing-list.bitcoindevs.xyz/bitcoindev`  

In Mailing List Summaries Repo: 
3. Fetch the documents for respective URLs provided in the list and perform the operations.

Comment (kouloumos) [reply generate_weekly_newsletter_json.py:23]: okay, thanks for the clarification. Can you remind me why we are doing this "domain injection"? Why not scrape directly from `https://mailing-list.bitcoindevs.xyz/bitcoindev`

Comment (urvishp80) [reply generate_weekly_newsletter_json.py:23]: Basically `https://gnusha.org/pi/bitcoindev` is our data source, and `https://mailing-list.bitcoindevs.xyz/bitcoindev` is our custom domain where we are managing [Bitcoin TLDR](https://tldr.bitcoinsearch.xyz/)

For more info refer: [this chat](https://discord.com/channels/1189471179967315968/1255834702972194827/1255858797923270796)

**Review by kouloumos — APPROVED** [review]
@urvishp80 it took me some time to approve this and it seems that there is a merge conflict now because of the cron-job generated files. After you resolve the conflict feel free to merge this.

**Pull request merged**